apps/music/test/marionette/Player_test.js

```diff
@@ -267,6 +267,37 @@ marionette('Music player tests', function() {
     });
   });
 
+  suite('Repeat test', function() {
+    test('Check that repeat button works', function() {
+      try {
+        music.launch();
+        music.waitForFirstTile();
+        music.switchToSongsView();
```

Comment:
I couldn't get `music.waitForListEnumerate` to work at all, so I didn't include it.